src/bp/ui-studio/src/web/views/FlowBuilder/diagram/index.tsx

```diff
@@ -167,36 +184,23 @@ export default class FlowBuilder extends Component<Props> {
     this.props.createFlow(name + '.flow.json')
   }
 
-  onDiagramDoubleClick = (event?: MouseEvent) => {
-    if (event) {
-      // We only keep 3 events for dbl click: full flow, standard nodes and skills. Adding temporarily router so it's editable
-      const target = this.diagramWidget.getMouseElement(event)
-      if (
-        target &&
-        !(
-          target.model instanceof StandardNodeModel ||
-          target.model instanceof SkillCallNodeModel ||
-          target.model instanceof RouterNodeModel
```

Comment:
This logic was there to prevent users from editing directly the properties of the V2 nodes, since they should only have one purpose (we hide the logic behind them)

Reply:
Also, it prevented user from selecting links and other elements, eg: 

![image](https://user-images.githubusercontent.com/42552874/66423774-57ff4400-e9da-11e9-9adf-f8ea2c4ed2f3.png)